test/unit/MatchingEngine.spec.ts

```diff
@@ -214,14 +216,14 @@ describe('MatchingEngine.removePeerOrders', () => {
     const secondHostId = 2;
 
     expect(engine.isEmpty()).to.be.true;
-    expect(engine.removePeerOrders(() => true)).to.be.empty;
+    expect(engine.removePeerOrders(0)).to.be.empty;
```

Comment:
this was meant to test removal of all orders. If you want to use `hostId` instead of a predicate function, you can remove this line.

Reply:
I can bring this back with another predicate defined outside of the tests, or I can create a `clearPeerOrders` method. But I also think it's OK to leave this test out since it's more a test of FastPriorityQueue than our code, at least until we start using a "clear" function on orders.

Reply:
Yes, if you remove the predicate function support, just remove this line.